src/index.js

```diff
@@ -7,6 +7,9 @@ import {css, styleSheet} from 'glamor'
 import shouldForwardProperty from './should-forward-property'
 
 const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+const domElements = htmlTagNames.concat(svgTagNames)
```

Comment:
To be clear, let's do that second option with the `filter` rather than messing with the reducer functions.